Source/extensions/processcontainers/common/BaseRefCount.h

```diff
@@ -19,6 +19,10 @@
 
 #pragma once
 
+#include <cstdint>
+
+#include "core/Sync.h"
```

Comment:
Do not include single header files from a module but always the whole module, which always has the name <modulename>/<modulename>.h which in this case would be "core/core.h".
I think the <cstdint> is aleready included here: https://github.com/rdkcentral/Thunder/blob/master/Source/core/Portability.h#L362 which is part of the core/core.h (and indicrectly through "core/Sync.h" so I do not htink it is needed and thus can be removed.

Reply:
As you mentioned, I tried to remove <cstdint> and changed to "core/core.h" but it was throwing error. So I removed <cstdint> and didn't change "core/Sync.h", then it was fine. Is it possible to do like this... if it is fine, can I push the changes to the code..

Reply:
I removed "cstdint" from BaseRefCount.h

Reply:
I think the error you got was a missing MODULE_NAME which is intentional as code (for the non-functional streams TRACING/LOGGING/WARNING REPORTING) need a source and the error complains it was not defined. What you could have done was #define MODULE_NAME Containers and than #include <core/core.h> by including the "core/Sync.h" the MODULE is now identified as "Core", which is technically incorrect as this is the Container module :-)   So could cou try again to include Core, but before including it set the right MODULE_NAME ? See: https://github.com/rdkcentral/Thunder/blob/master/Source/extensions/processcontainers/Module.h
Just learned that the Module file for this artifact already exists, I suggers including this Module file like: #include "../Module.h" i.s.o. "core/Sync.h"